client/src/game/ui/HanabiCardInit.ts

```diff
@@ -502,9 +498,10 @@ export function empathy(this: HanabiCard) {
     setEmpathyOnHand(true);
   });
 
-  // Konva.PointerEvent does not have a "type" property for some reason
-  this.on('mouseup mouseout', (event: any) => {
-    if (event.type === 'mouseup' && event.evt.which !== 1) { // Left-click
+  this.on('mouseup mouseout', (event: Konva.KonvaEventObject<MouseEvent>) => {
+    // Konva.MouseEvent does not have a "type" property for some reason
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
did you test that this works?

Reply:
Yep, empathy is still working, and you can drag the mouse out to cancel. I still recommend you checkout the branch and do a pass since you know the details better.